File: t/VB/RasterizeDocumentDemo/UserControls/XpsOptionsControl.vb
' *************************************************************
' Copyright (c) 1991-2019 LEAD Technologies, Inc.              
' All Rights Reserved.                                         
' *************************************************************
Imports Leadtools
Imports Leadtools.Codecs

Public Class XpsOptionsControl
   Implements IOptionsUserControl

   ''' <summary>
   ''' Called by the owner to initialize
   ''' </summary>
   Public Sub SetData(ByVal rasterCodecsInstance As RasterCodecs) Implements IOptionsUserControl.SetData

      ' Set the state of the controls

      Dim xpsLoadOptions As CodecsXpsLoadOptions = rasterCodecsInstance.Options.Xps.Load
   End Sub

   ''' <summary>
   ''' Called by the owner to get the data
   ''' </summary>
   Public Function GetData(ByVal rasterCodecsInstance As RasterCodecs) As Boolean Implements IOptionsUserControl.GetData
      Dim xpsLoadOptions As CodecsXpsLoadOptions = rasterCodecsInstance.Options.Xps.Load

      Return True
   End Function
End Class

File: t/CS/RasterizeDocumentDemo/UserControls/XpsOptionsControl.cs
// *************************************************************
// Copyright (c) 1991-2019 LEAD Technologies, Inc.              
// All Rights Reserved.                                         
// *************************************************************
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Drawing;
using System.Data;
using System.Text;
using System.Windows.Forms;

using Leadtools;
using Leadtools.Codecs;

namespace RasterizeDocumentDemo.UserControls
{
   public partial class XpsOptionsControl : UserControl, IOptionsUserControl
   {
      public XpsOptionsControl()
      {
         InitializeComponent();
      }

      /// <summary>
      /// Called by the owner to initialize
      /// </summary>
      public void SetData(RasterCodecs rasterCodecsInstance)
      {
         // Set the state of the controls

         CodecsXpsLoadOptions xpsLoadOptions = rasterCodecsInstance.Options.Xps.Load;
      }

      /// <summary>
      /// Called by the owner to get the data
      /// </summary>
      public bool GetData(RasterCodecs rasterCodecsInstance)
      {
         CodecsXpsLoadOptions xpsLoadOptions = rasterCodecsInstance.Options.Xps.Load;

         return true;
      }
   }
}
